pkg/acme/webhook/registry/challengepayload/challenge_payload.go

```diff
@@ -46,6 +47,9 @@ func NewREST(hookFn webhook.Solver) *REST {
 func (r *REST) New() runtime.Object {
 	return &v1alpha1.ChallengePayload{}
 }
+func (r *REST) GetSingularName() string {
+	return "ChallengePayload"
```

Comment:
This name does not matter and will never be used as there will never be ChallengePayload resources in cluster